tests/tensor/rewriting/test_basic.py

```diff
@@ -861,7 +863,7 @@ def test_local_mul_switch_sink(self):
                 f = self.function_remove_nan(
                     [condition[0], x[0], c], [y], mode=self.mode
                 )
-                if type(condition[1]) is list:
+                if isinstance(condition[1], list):
```

Comment:
Was this change automatic or by hand? I think this is safe, but strictly speaking `condition[1]` could theoretically be a subclass of `list` (although subclassing builtin types is generally pretty broken, so nobody should ever do it). The reason I bring it up is because we're a bit stalled on #581 due to similar issues. This PR will make that obsolete, but I'm curious if you have any thoughts.